apiextensions/storageversion/migrator.go

```diff
@@ -72,7 +72,8 @@ func (m *Migrator) Migrate(ctx context.Context, gr schema.GroupResource) error {
 	}
 
 	patch := `{"status":{"storedVersions":["` + version + `"]}}`
-	_, err = crdClient.Patch(crd.Name, types.StrategicMergePatchType, []byte(patch), "status")
+	_, err = crdClient.
+		Patch(ctx, crd.Name, types.StrategicMergePatchType, []byte(patch), metav1.PatchOptions{}, "status")
```

Comment:
Nit: Maybe make this one line again?

Reply:
done in 77678bb83f220dd422bf886e7ce009f6f23d6616